Common/Orders/OrderTicket.cs

```diff
@@ -162,7 +162,13 @@ public IReadOnlyList<UpdateOrderRequest> UpdateRequests
         /// </summary>
         public CancelOrderRequest CancelRequest
         {
-            get { return _cancelRequest; }
+            get
+            {
+                lock (_cancelRequestLock)
+                {
+                    return _cancelRequest;
```

Comment:
There's never a need to lock around a return statement like this since returning an object reference is an atomic operation, IOW, it's impossible for this line to be corrupted by a race condition. No absolute need to change this, but figured I'd add some technical detail.